xchange-btcturk/src/main/java/org/knowm/xchange/btcturk/dto/trade/BTCTurkOrder.java

```diff
@@ -18,17 +18,37 @@
     private String TriggerPrice;
     private String TriggerPricePrecision;
     private BTCTurkOrderTypes OrderType;
-    private CurrencyPair PairSymbol;
-    private String DateTime;
+    private BTCTurkPair PairSymbol;
+
+    public BTCTurkOrder(
+  	@JsonProperty("OrderMethod") BTCTurkOrderMethods OrderMethod,
+  	@JsonProperty("Price") String Price,
+  	@JsonProperty("PricePrecision") String PricePrecision,
+  	@JsonProperty("Amount") String Amount,
+  	@JsonProperty("AmountPrecision") String AmountPrecision,
+  	@JsonProperty("Total") String Total,
+  	@JsonProperty("TotalPrecision") String TotalPrecision,
+  	@JsonProperty("DenominatorPrecision") int DenominatorPrecision,
+  	@JsonProperty("TriggerPrice") String TriggerPrice,
+  	@JsonProperty("TriggerPricePrecision") String TriggerPricePrecision,
+  	@JsonProperty("OrderType") BTCTurkOrderTypes OrderType,
+  	@JsonProperty("PairSymbol") BTCTurkPair PairSymbol
+  			  ){
+  	this.OrderMethod = OrderMethod;	
+  	this.Price = Price;	 
+  	this.PricePrecision = PricePrecision;	 
+  	this.Amount = Amount;	 
+  	this.AmountPrecision = AmountPrecision;	 
+  	this.Total = Total;	 
+  	this.TotalPrecision = TotalPrecision;	 
+  	this.DenominatorPrecision = DenominatorPrecision;	 
+  	this.TriggerPrice = TriggerPrice;	 
+  	this.TriggerPricePrecision = TriggerPricePrecision;	 
+  	this.OrderType = OrderType;	
+  	this.OrderType = OrderType;	 
+  	this.PairSymbol = PairSymbol;	 
+  	  }
```

Comment:
formatting in this file and in many others is broken
maven is setup to fix formatting in every file
how did you build this project? 
I do not see how this would get pass `mvn install`